website: plannable import docs (#33266)
hashicorp/terraform: website/docs/cli/import/index.mdx

Terraform can import existing infrastructure resources. This functionality lets you bring existing resources under Terraform management.

> -> **Note:** Terraform v1.5.0 and later supports `import` blocks. Unlike the `terraform import` command, you can use `import` blocks to import more than one resource at a time, and you can review imports as part of your normal plan and apply workflow. [Learn more about `import` blocks](/terraform/language/import).

Reviewer: I think we should leave the config generation note to that page to keep this note as short and specific as possible; but don't feel super strongly about it.

> -> **Note:** Terraform v1.5.0 and later supports the use of `import` blocks in code. Unlike the `terraform import` command, `import` blocks can be used to import more than one resource at a time, and imports can be reviewed as part of a normal plan and apply workflow. Refer to [Language -> Import](/terraform/language/import) for more details.

Reviewer: Suggestions for active voice and brevity:
```suggestion
-> **Note:** Terraform v1.5.0 and later supports the use of `import` blocks in code. Unlike the `terraform import` command, you can use `import` blocks to import more than one resource at a time, and you can review imports as part of your normal plan and apply workflow. [Learn more about Import](/terraform/language/import).
```

## State Only

~> **Warning:** Terraform expects that each remote object is bound to a _single_ resource address. You should import each remote object to _one_ Terraform resource address. If you import the same object multiple times, Terraform may exhibit unwanted behavior. See [State](/terraform/language/state) for more details.

> Terraform import can only import resources into the [state](/terraform/language/state). Importing via the CLI does not generate configuration. You can use `import` blocks to generate configuration.

Reviewer 2: > Terraform import can only import resources into the [state]

swap -> "Terraform `import` CLI command can only import resources into state...."
